lib/search/presenters/result_set_presenter.rb

```diff
@@ -37,7 +37,7 @@ def present
         results: presented_results,
         total: es_response["hits"]["total"],
         start: search_params.start,
-        facets: presented_facets,
+        aggregates: presented_aggregates,
```

Comment:
We are preserving backwards compatibility for facets in the request params, but this won't work if we remove `facets` from the response.

Assuming we are going ahead with the rename, I think we should make it return `facets` if you ask for facets and `aggregates` if you ask for aggregates (and just reject requests that mix the two).